conans/client/tools/scm.py

```diff
@@ -176,11 +177,31 @@ def runner_no_strip(command):
         runner = runner or runner_no_strip
         super(SVN, self).__init__(folder=folder, runner=runner, *args, **kwargs)
 
+        try:
+            self.version = SVN.get_version()
+        except ConanException:
+            self.version = Version("1.10")  # TODO: Go for a modern one, or raise?
```

Comment:
@lasote, what would you do here? What to do if the function to parse SVN version fails?